WeakAuras/Transmission.lua

```diff
@@ -321,9 +332,12 @@ local function install(data, oldData, patch, mode, isParent)
     if data.uid and data.uid ~= oldData.uid then
       oldData.uid = data.uid
```

Comment:
Doesn't this need code to adjust the UID to ID mapping?

Reply:
done